apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js

```diff
@@ -54,13 +53,13 @@ describe("DetailViewContents", () => {
     });
 
     it("Uses entered value for facilitator applications with notes", () => {
-      const facilitatorDetailView = mountDetailView('Facilitator', {notes: "actual notes"});
-      expect(facilitatorDetailView.state().notes).to.eql("actual notes");
+      const teacherDetailView = mountDetailView('Facilitator', {notes: "actual notes"});
+      expect(teacherDetailView.state().notes).to.eql("actual notes");
     });
```

Comment:
this one should stay `facilitatorDetailView` because it is testing a facilitator application.